apps/preciousmetals/precious_metals.star

```diff
@@ -0,0 +1,56 @@
+"""
+Applet: Precious Metals
+Summary: Quotes on precious metals
+Description: Quotes for gold, platinum and silver.
+Author: threeio
+"""
+
+load("render.star", "render")
+load("http.star", "http")
+load("encoding/base64.star", "base64")
+
+METALS_PRICE_URL = "https://api.metals.live/v1/spot"
+
+IMAGE = base64.decode("""iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAAXNSR0IArs4c6QAAAHhJREFUOE+1k7sNwDAIRPGojBB5DisjMGoiCiTCJyJGoXHhu2dOhgHNGk0/pABEvCyciJw+BLB5HtM1t84FFuIAmVloFvIAWDOLpXRHGpICWBTFYGAJIEI+LagM0Lm3IkT5XyPwZesX5MXWHGjI9iR+2Y//lqnaxQ21HFYRgy5eOgAAAABJRU5ErkJggg==""")
+
+load("schema.star", "schema")
+
+def main(config):
+    rep = http.get(METALS_PRICE_URL)
```

Comment:
It would be good to add some caching here. Even if it's only a few minutes, it will greatly reduce the number of API calls to this service.

Reply:
I'll work on it for v1.1 ;)